cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControl.java

```diff
@@ -651,6 +651,8 @@ private static boolean hasProposalsToExecute(Collection<ExecutionProposal> propo
    *                                (if null, use default.replica.movement.strategies).
    * @param replicationThrottle The replication throttle (bytes/second) to apply to both leaders and followers
```

Comment:
Let's clarify in the comment that replicationThrottle is for inter-broker, logDirThrottle is for intra-broker rebalance.

Reply:
Fixed. I elaborated the Javadoc by explicitly stating inter-broker movements vs. intra-broker movements.